src/logging.rs

```diff
@@ -292,6 +298,13 @@ impl Server {
         }
     }
 
+    pub fn with_tls(self, tls: TlsStatus) -> Self {
```

Comment:
Please name this `with_tls_status`. I think the name `with_tls` is misleading.

Reply:
Done in 710b54ac7922793eeba7939ab7ca1c03b0822f0f. You're right that the old name was potentially confusing.